### DISCUSSION TITLE
[Query] FeaturesToEnable in Google Photos Patch missing some Pixel XL features.

### DISCUSSION THREAD

**Discussion by archit9169** [ReVanced/revanced-discussions#1063] [Question] [answered]
Hi, I was wondering if someone can explain why some feature options are missing from FeatureToEnable list in Google Photos patch.

Specifically ->
"com.google.android.feature.PIXEL_EXPERIENCE",
"com.google.android.apps.photos.PIXEL_PRELOAD",
"com.google.android.apps.photos.PIXEL_2016_PRELOAD"

These 3 are missing from the list which are for Pixel XL feature list.

My question is:
1. Are they not needed?
2. What are they specifically used for?
3. Any additional info that someone can give for better understanding of this.

**Comment by oSumAtrIX** [comment, marked as answer]
The patch replaces existing strings in the Photos app. The strings you mentioned do not exist in the app so therefore it doesn't make sense to look for them and replace them.

**Reply by archit9169** [reply]
Thanks for the confirmation. Also, another question -> what will happen if these strings are added to the app through the patch? I have been using pixel patches in custom roms for quite some time, and I have always added these 3 features to which ever rom I was using. Was these not even needed for pixel features to work?

**Reply by oSumAtrIX** [reply]
If the string's don't exist in the app in one form or another, adding them has no effect.
